uri.gemspec

```diff
@@ -12,13 +12,21 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{URI is a module providing classes to handle Uniform Resource Identifiers}
   spec.description   = spec.summary
-  spec.homepage      = "https://github.com/ruby/uri"
+
+  github_link        = "https://github.com/ruby/uri"
+
+  spec.homepage      = github_link
   spec.licenses      = ["Ruby", "BSD-2-Clause"]
 
   spec.required_ruby_version = '>= 2.4'
 
-  spec.metadata["homepage_uri"] = spec.homepage
-  spec.metadata["source_code_uri"] = spec.homepage
+  spec.metadata = {
+    "bug_tracker_uri" => "#{github_link}/issues",
+    "changelog_uri" => "#{github_link}/releases",
+    "documentation_uri" => "https://rubydoc.info/gems/uri",
```

Comment:
```suggestion
    "documentation_uri" => "https://ruby.github.io/uri/",
```

Reply:
No problem, thanks. Also changed in the README.